src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -969,8 +969,8 @@ int fx_muxer_t::handle_cli(
     // Did not exececute the app or run other commands, so try the CLI SDK dotnet.dll
     //
 
-    pal::string_t sdk_dotnet;
-    if (!sdk_resolver_t::resolve_sdk_dotnet_path(host_info.dotnet_root, &sdk_dotnet))
+    auto sdk_dotnet = sdk_resolver::from_nearest_global_file().resolve(host_info.dotnet_root);
```

Comment:
Does this work when the SDK/SharedHost is installed and registered to a non-programfiles install location?

Reply:
This is no change in the current behavior.  This will use a SDK resolver based on the nearest global.json from CWD and use `host_info.dotnet_root` as the dotnet install root for resolving SDKs.  So if `host_info.dotnet_root` points at the non-program files location (either via environment or set from where `dotnet` is executing from), then we'll resolve SDKs from that install location.